dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/error/FallbackExceptionHandler.java

```diff
@@ -45,8 +45,8 @@ public void handle() throws FacesException {
             if (facesContext.getPartialViewContext().isAjaxRequest()) {
                 JsfHelper.addErrorMessage(null, BundleUtil.getStringFromBundle("error.general.message"), "");
             } else {
-                NavigationHandler nav = facesContext.getApplication().getNavigationHandler();
-                nav.handleNavigation(facesContext, null, "/500");
+                Try.run(() -> facesContext.getExternalContext().redirect("500.xhtml"))
+                        .onFailure(exception -> logger.log(Level.SEVERE, exception.getMessage(), exception));
```

Comment:
In what cases this worked incorrectly?
There is a difference between redirect and navigate. I'm not sure if we want to redirect (it will change url in browser to 500.xhtml)

Reply:
It worked incorrectly when you tried to access the page and exception was thrown right away. If we don't want 500 in url, it will probably also work with navigation but with added ?faces-redirect=true.